src/pages/term/Term.module.sass

```diff
@@ -0,0 +1,6 @@
+.button-list
+    display: flex
+    justify-content: space-between
+
+.after-markdown
+    margin-block-start: 2em
```

Comment:
- This class is above the previous in the TSX hierarchy, but not in the Sass.
- There isn't a need in this class it can be combined with the above.
  Add the margin to the buttons group instead.

Reply:
We might add other stuff down there in the future. The margin is for distinguishing the markdown file from additional content and it's not specific to the button group